flows/index.py

```diff
@@ -1022,6 +1083,7 @@ async def index_labelled_passages_from_s3_to_vespa(
     classifier_specs: list[ClassifierSpec] | None = None,
     document_ids: list[str] | None = None,
     config: Config | None = None,
+    change: Change | None = Change.ADD_AND_UPDATE,
```

Comment:
suggestion: Could we just implement a separate flow for classifier deletion? Judging by how deep we are passing the change param down, it could make our lives easier to just have a separate classifier removal flow that removals all occurrences from the concepts field of the document passage table and the concept counts field of the family table?

Reply:
That's how I originally started, with attempt no. 1, then I realised I can reuse all this work. I have thought about refactoring what would be shared, but more focused on getting something working. That could be done later.

Reply:
It's back to a separate, new pipeline!